modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -492,17 +499,65 @@ void cv::gimpl::ie::GIEExecutable::run(std::vector<InObj>  &&input_objs,
 namespace cv {
 namespace gimpl {
 namespace ie {
+inline void configureReshapeByInputImage(const IE::InputInfo::Ptr& ii,
+                                         const IEUnit& uu,
+                                         const std::vector<int>& image_size) {
+    // Name for current layer
+    const auto layer_name = ii->name();
+    // Checking that params contains this name
```

Comment:
Sorry for picking on little things, but it isn't a `Checking`,  you do checking below.

Reply:
Changed.